BUCK

```diff
@@ -0,0 +1,22 @@
+genrule(
```

Comment:
how can I access files in the project-root like LICENSE from scripts/packages/debian ?